src/main/resources/react/src/components/local/ru.js

```diff
@@ -16,13 +16,35 @@ const ru = {
 		logEmail: 'Введите верный логин или email',
 		logPassword: 'Неверный пароль',
 		regEmail: 'Укажите действующий адрес электронной почты',
-		regNick: 'Больше 2 и меньше 50 символов, без пробелов',
+		regNick: 'Больше 1 и меньше 50 символов, без пробелов',
```

Comment:
The same question as en.js has [here](https://github.com/Jack11M/EVO-Exchange-BE-2019/pull/175/files#r604223230)